dev/validate_bom__config.py

```diff
@@ -1194,6 +1383,20 @@ def make_scripts(options):
   return init_script, config_script
 
 
+def setup_environments(options):
```

Comment:
should this be singular "setup_environment"?
same for teardown below

Reply:
Done.